common/common-api/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
     compile group: 'org.jasypt', name: 'jasypt', version: '1.9.3'
 
     testRuntime 'org.jruby:jruby-complete:9.0.5.0'
-    testRuntime 'org.python:jython-standalone:2.7.0'
+    testRuntime 'org.python:jython-standalone:2.7.1'
```

Comment:
We cannot update jython, this has caused many issues when we tried to do this in the past